Feature/layout

Poldyaich97 opened the pull request:
add Reviewers

Poldyaich97 committed:
remove unnecessary files

Poldyaich97 committed:
add Header

Poldyaich97 committed:
Added a logo.svg
Added a search.svg
I wrote the header.

Poldyaich97 committed:
Fixed

Poldyaich97 committed:
added Main,
fixed Header,
added a block explore in main

Poldyaich97 committed:
add block experiences

Poldyaich97 committed:
Main "done"

Poldyaich97 committed:
add footer.
fixed main

Commit:
Исправил header

Commit:
fix Header.jsx

Commit:
Header done

Commit:
header  fix and done

Commit:
add Explore Airbnb

Commit:
add Experiences

Commit:
add column

Poldyaich97 committed:
fix 12122021

Commit:
add featured

Commit:
Add footer

Commit:
fix

Commit:
fix

Commit:
add arrow

Poldyaich97 committed:
Fix styled main and explore

Commit:
add Stub

Poldyaich97 committed:
Fix RaitingLine.
Fixed the indentation

Poldyaich97 committed:
fix

chervyakovru commented on src/App.jsx:
```diff
@@ -0,0 +1,40 @@
+const AppHeader = styled(Header)`
+  margin-bottom: 48px;
+`;
+const AppMain = styled(Main)``;
```
Можно убрать  styled

chervyakovru commented on src/App.jsx:
```diff
@@ -0,0 +1,40 @@
+  }
+`;
+
+const StyledStub = styled(Stub)`
```
Будет проще, Если Stub По умолчанию будет отображаться, а медиа выражением его скрывать на десктопе

chervyakovru replied:
Это опять же соответствует идее компонентного подхода

chervyakovru replied:
И тогда врапер будет не нужен

chervyakovru commented on src/Fonts/@font-face/Demo.html:
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```
Этот файл не нужен

chervyakovru replied:
Все еще не нужен

chervyakovru commented:
Можно убрать папку @font-face

chervyakovru commented on src/Footer/Footer.jsx:
```diff
@@ -0,0 +1,213 @@
+import twitter from "./twitter.svg";
+import facebook from "./facebook.svg";
+
+const StyledFooter = styled.div``;
```
styled не нужен

chervyakovru commented on src/Footer/Footer.jsx:
```diff
@@ -0,0 +1,213 @@
+  width: 228px;
+  border: 1px solid rgba(72, 72, 72, 0.2);
+  border-radius: 4px;
+  margin-bottom: 16px;
```
margin-bottom лучше убрать, добавить flex column-gap в StyledBurger

Poldyaich97 replied:
Почему убрать? это же отталкивает один select от другого
![image](https://user-images.githubusercontent.com/55269933/149924944-66d04a07-fb18-4fdb-98db-2eda5b73adb0.png)

Poldyaich97 replied:
вопрос отпал

chervyakovru replied:
А коммент в коде остался

chervyakovru commented:
![image](https://user-images.githubusercontent.com/14909082/149662719-d679427a-3b5e-4cd1-abf2-cca2fc99e5b6.png)
![image](https://user-images.githubusercontent.com/14909082/149662746-0cbcfc67-68e5-4490-921b-bb07be740d8b.png)

Проект не запускается. Названия папок не соответствует

chervyakovru commented on src/Footer/Footer.jsx:
```diff
@@ -0,0 +1,213 @@
+  font-size: 14px;
+  line-height: 22px;
+  margin: 0;
+  padding: 0;
```
padding лишний

Poldyaich97 replied:
done

chervyakovru commented on src/Footer/Footer.jsx:
```diff
@@ -0,0 +1,213 @@
+`;
+const StyledRight = styled.div`
+  margin-left: auto;
+  margin-right: 0;
```
margin-right: 0; не нужен

chervyakovru commented on src/Footer/facebook.svg:
```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```
![image](https://user-images.githubusercontent.com/14909082/149663238-9c25300c-4876-407b-8b22-179953d905c5.png)
![image](https://user-images.githubusercontent.com/14909082/149663240-e004b1bb-fa65-4b9b-8d78-fee571381bd9.png)


![image](https://user-images.githubusercontent.com/14909082/149663257-d0b77d03-b345-438f-95cf-7cf3bf0f3b4f.png)


Иконки лучше выгружать одного размера - так макет будет ровным и не нужен будет подгонять отступы чтобы выглядело как на макете. А также таким образом увеличивается клиекабельная область и пользователю проще нажимать

chervyakovru commented on src/Footer/footerLogo.svg:
```diff
@@ -0,0 +1,3 @@
+<svg width="21" height="22" viewBox="0 0 21 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Ресурсы лучше называть по смыслу, а не по назначению. Не footerLogo, а blackWhiteLogo

chervyakovru replied:
Остался footerLogo

Poldyaich97 replied:
done

chervyakovru commented on src/Header/Header.jsx:
```diff
@@ -0,0 +1,39 @@
+`;
+const StyledImg = styled.img`
+  width: 30px;
+  height: 30px;
```
![image](https://user-images.githubusercontent.com/14909082/149663450-dc5f812d-6a3b-4c55-9eaf-7a6498822f91.png)
![image](https://user-images.githubusercontent.com/14909082/149663455-10561fff-d6cb-4de2-824b-a38593b832d9.png)

На самом деле лого не квадратный

Poldyaich97 replied:
done

chervyakovru commented on src/Header/Header.jsx:
```diff
@@ -0,0 +1,39 @@
+            <StyledImg src={logo} alt="logo" />
+          </Logo>
+          <Search />
+          <Nav />
```
Кажется margin-left остался в компоненте

Poldyaich97 replied:
тут не совсем понял?

chervyakovru commented on src/Header/Nav.jsx:
```diff
@@ -0,0 +1,40 @@
+  font-size: 14px;
+  line-height: 24px;
+  text-decoration: none;
+  color: black;
```
кажется тут не black

Poldyaich97 replied:
done

chervyakovru commented on src/Header/Nav.jsx:
```diff
@@ -0,0 +1,40 @@
+  line-height: 24px;
+  text-decoration: none;
+  color: black;
+  padding: 12px 0px;
```
не сработали padding потому что display: inline

Poldyaich97 replied:
done

chervyakovru commented on src/Header/Nav.jsx:
```diff
@@ -0,0 +1,40 @@
+  text-decoration: none;
+  color: black;
+  padding: 12px 0px;
+  font-weight: 400;
```
кажется такой вес по умолчанию

Poldyaich97 replied:
done

Poldyaich97 committed:
fixed footer, add instagram

chervyakovru commented:
![image](https://user-images.githubusercontent.com/14909082/149663750-3e20791e-e09e-4901-a1fe-c530a69d1655.png)

На импуте сломался шрифт

chervyakovru commented on src/Main/Experiences/Experiences.jsx:
```diff
@@ -0,0 +1,111 @@
+  column-gap: 17px;
+  overflow: hidden;
+`;
+const HeaderTitle = styled.div`
```
Header Title можно весь вынести в компонент чтобы seeAll каждый раз не писать

Poldyaich97 replied:
возможно, но seeall используется в 3 случаях  из 4

Poldyaich97 replied:
Мне кажется, что лучше  дополнительно его докидывать

chervyakovru commented on src/Main/Experiences/Experiences.jsx:
```diff
@@ -0,0 +1,111 @@
+  line-height: 1;
+  margin-bottom: 6px;
+`;
+const TittleTitle = styled.h5`
```
do u mean subTitle?)

Poldyaich97 replied:
done

chervyakovru commented:
#пиксельдрочево

![image](https://user-images.githubusercontent.com/14909082/149663996-285f5b69-dc74-4733-8575-712fe3b00a7c.png)

chervyakovru commented on src/Main/Experiences/Experiences.jsx:
```diff
@@ -0,0 +1,111 @@
+`;
+const Columns = styled.div`
+  display: flex;
+  column-gap: 17px;
```
Скорее всего тут также 18px как между всеми остальными колонками

Poldyaich97 replied:
по макету было 17. исправил

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Experiences/Experiences.jsx:
```diff
@@ -0,0 +1,111 @@
+const Columns = styled.div`
+  display: flex;
+  column-gap: 17px;
+  overflow: hidden;
```
зачем тут overflow?

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Explore/Explore.jsx:
```diff
@@ -0,0 +1,77 @@
+  margin-right: 24px;
+`;
+const StyledImg = styled.img`
+  height: 100%;
```
тут хватит только height: 100%

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Explore/Explore.jsx:
```diff
@@ -0,0 +1,77 @@
+  display: flex;
+  align-items: center;
+`;
+const StyledDescription = styled.p`
```
Можно не писать Styled. Просто Description

Poldyaich97 replied:
done

chervyakovru commented:
Растровые картинки нельзя выгружать в svg формате, Они хуже адаптируются, практически не сжимаются и дольше рендерятся. А также стоит называть картинки по смыслу, а не по месту использования

`columnsTwo.svg` -> experiences.jpg

chervyakovru commented on src/Main/Featured/Featured.jsx:
```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+import { StyledTitle } from "../SectionTitle/SectionTitle";
+import featuredOne from "./Featured.png";
+const StyledFeatured = styled.div``;
```
не нужен styled

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Featured/Featured.jsx:
```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+import { StyledTitle } from "../SectionTitle/SectionTitle";
+import featuredOne from "./Featured.png";
```
featuredOne -> featured
Названия ресурсов лучше писать строчными буквами

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Featured/Featured.jsx:
```diff
@@ -0,0 +1,50 @@
+
+const Columns = styled.div`
+  display: flex;
+  column-gap: 17px;
```
18

chervyakovru commented on src/Main/Featured/Featured.jsx:
```diff
@@ -0,0 +1,50 @@
+const Columns = styled.div`
+  display: flex;
+  column-gap: 17px;
+  overflow: hidden;
```
зачем hidden

chervyakovru commented on src/Main/Homes/Homes.jsx:
```diff
@@ -0,0 +1,77 @@
+const Columns = styled.div`
+  display: flex;
+  column-gap: 17px;
+  overflow: hidden;
```
18 overflow

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Homes/Homes.jsx:
```diff
@@ -0,0 +1,77 @@
+  justify-content: space-between;
+  margin-bottom: 24px;
+`;
+const Column = styled.div``;
```
styled

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Homes/Homes.jsx:
```diff
@@ -0,0 +1,77 @@
+const PriceTitle = styled.div`
+  font-weight: bold;
+`;
+const TittleTitle = styled.h5`
```
subTitle

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Homes/Homes.jsx:
```diff
@@ -0,0 +1,77 @@
+const NameSubtitle = styled.p`
+  margin: 0;
+  font-weight: 400;
+  line-height: 18px;
```
Возможно во всем документе по умолчанию стоит сделать line-height 18?

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Main.jsx:
```diff
@@ -0,0 +1,40 @@
+  margin-bottom: 48px;
+`;
+const StyledExperiences = styled(Experiences)`
+  margin-bottom: 68px;
```
На самом деле между всеми ними по 48 px

Poldyaich97 replied:
done row-gap

chervyakovru commented on src/Main/Popular/Popular.jsx:
```diff
@@ -0,0 +1,74 @@
+  display: flex;
+  justify-content: space-between;
+`;
+const Column = styled.div``;
```
styled

Poldyaich97 replied:
div

chervyakovru commented on src/Main/Popular/Popular.jsx:
```diff
@@ -0,0 +1,74 @@
+  display: block;
+  font-size: 12px;
+  line-height: 12px;
+  margin: 0px;
```
TT: uppercase

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Popular/Popular.jsx:
```diff
@@ -0,0 +1,74 @@
+`;
+const СountryTitle = styled.h5`
+  display: block;
+  font-size: 12px;
```
fz: 10

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Popular/Popular.jsx:
```diff
@@ -0,0 +1,74 @@
+  margin: 0;
+  font-size: 18px;
+  line-height: 21px;
+  font-weight: 400;
```
light

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Raiting/RatingLine.jsx:
```diff
@@ -0,0 +1,37 @@
+`;
+export default function SeeAll() {
+  return (
+    // <React.Fragment>
```
remove comment

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Raiting/RatingLine.jsx:
```diff
@@ -0,0 +1,37 @@
+  width: 12px;
+  height: 12px;
+`;
+const ReviewsLine = styled.div`
```
styled не нужкен

Poldyaich97 replied:
done

chervyakovru commented on src/Main/Raiting/RatingLine.jsx:
```diff
@@ -0,0 +1,37 @@
+  column-gap: 8px;
+`;
+const RaitingLine = styled.div`
+  text-align: start;
```
text-alight нужен?

Poldyaich97 replied:
done

chervyakovru commented:
![image](https://user-images.githubusercontent.com/14909082/149665005-e6306f84-4df4-44c7-835c-c0d3e9fb0308.png)

Картинка уже чем колонка

chervyakovru commented on src/Main/Raiting/RatingLine.jsx:
```diff
@@ -0,0 +1,37 @@
+const ReviewsLine = styled.div`
+  margin: 0;
+`;
+export default function SeeAll() {
```
Кажется это не SeeAll

Poldyaich97 replied:
done

chervyakovru commented:
Есть 2 разных рейтинга. У тебя это 1 компонент

![image](https://user-images.githubusercontent.com/14909082/149665033-e5fb891e-4bdb-46eb-a750-e442315468c0.png)

chervyakovru commented on src/Main/SectionSee/SectionSee.jsx:
```diff
@@ -0,0 +1,30 @@
+    z-index: 12;
+  }
+`;
+export default function SeeAll() {
```
лучше чтобы название файла и компоненте совпадали

Poldyaich97 replied:
SectionSee

Poldyaich97 replied:
done

chervyakovru commented on src/Main/SectionSee/SectionSee.jsx:
```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+import Arrow from "./arrow.svg";
+const StyledSeeAll = styled.div`
```
зачем нужен этот div?

Poldyaich97 replied:
done

chervyakovru commented:
![image](https://user-images.githubusercontent.com/14909082/149665171-194eb5f7-5684-4273-ba47-4b2217988d91.png)
![image](https://user-images.githubusercontent.com/14909082/149665173-bd7b1163-0c26-4531-ba4d-4a135ad08302.png)

Отличается макет от верстки

chervyakovru commented on src/Main/SectionSee/SectionSee.jsx:
```diff
@@ -0,0 +1,30 @@
+    position: absolute;
+    top: 0px;
+    left: 45px;
+    z-index: 12;
```
почему 12? можно 1

Poldyaich97 replied:
done

chervyakovru commented on src/Main/SectionSee/SectionSee.jsx:
```diff
@@ -0,0 +1,30 @@
+    content: url(${Arrow});
+    position: absolute;
+    top: 0px;
+    left: 45px;
```
проще right: 0

chervyakovru commented on src/Stub.jsx:
```diff
@@ -0,0 +1,58 @@
+
+const BackgroundStub = styled.div`
+  height: 100vh;
+  width: 100vw;
```
лучше задать в %. Если в 100vh, то будет скролится криво на мобилке

Poldyaich97 replied:
в процентах наоборот съезжает..

chervyakovru commented on src/Stub.jsx:
```diff
@@ -0,0 +1,58 @@
+  background-image: url(${bgStub});
+  background-size: cover;
+  background-position: center center;
+  backdrop-filter: blur(5px);
```
![image](https://user-images.githubusercontent.com/14909082/149665537-0f3e1421-b915-4e95-afa1-90af089615d3.png)
https://caniuse.com/?search=backdrop-filter

У backdrop-filter пока плохая поддержка. Лучше использовать что-то по проще

Poldyaich97 replied:
![image](https://user-images.githubusercontent.com/55269933/149952414-c4a2b6a1-db45-4609-a132-add3e4771bde.png)
 DONE

chervyakovru commented on src/Stub.jsx:
```diff
@@ -0,0 +1,58 @@
+  font-size: 24px;
+  font-weight: 600;
+  color: #f5f5f5;
+  font-family: "Circular Air Light";
```
Circular Air Light - не загружается такой шрифт

Poldyaich97 replied:
понять не могу почему у меня по всему проекту не работает шрифт.

chervyakovru commented on src/Stub.jsx:
```diff
@@ -0,0 +1,58 @@
+  font-weight: 600;
+  color: #f5f5f5;
+  text-align: center;
+  font-family: "Circular Air Light";
```
И тут

Commit:
fix

Commit:
fix fonts

chervyakovru commented:
> ![image](https://user-images.githubusercontent.com/14909082/149665005-e6306f84-4df4-44c7-835c-c0d3e9fb0308.png)
> 
> Картинка уже чем колонка

До сих пор уже

chervyakovru commented:
> Можно убрать папку @font-face

Все еще можно убрать

chervyakovru commented:
> ![image](https://user-images.githubusercontent.com/14909082/149662719-d679427a-3b5e-4cd1-abf2-cca2fc99e5b6.png) ![image](https://user-images.githubusercontent.com/14909082/149662746-0cbcfc67-68e5-4490-921b-bb07be740d8b.png)
> 
> Проект не запускается. Названия папок не соответствует

Все еще папка называется с большой буквы
<img width="1904" alt="Screenshot 2022-01-18 at 21 35 21" src="https://user-images.githubusercontent.com/14909082/149998325-a3524380-860f-4aee-a62a-2bf2ea64957b.png">

Commit:
Fix

Commit:
test

Commit:
test-2

Commit:
fix

Poldyaich97 committed:
fix

Commit:
все сломалось

Poldyaich97 committed:
fix

Poldyaich97 committed:
add react route

Commit:
add Hosts and data

Commit:
Сделал пока без объекта.
При нажатии на ссыли - меняется h3.
Но нужно, чтоб значения получались из объекта.. ищу решение

Commit:
добавил отступы

Commit:
Fix css
